web-ui/app/js/account_recovery/page.js

```diff
@@ -3,22 +3,22 @@ import { translate } from 'react-i18next'
 
 import 'scss/account_recovery/page.scss'
 
-const Page = (props) => (
+const Page = ({ t }) => (
   <div className='container'>
-    <img src='assets/images/forgot-my-password.svg' alt="Esqueci minha senha!"/>
+    <img src='assets/images/forgot-my-password.svg' alt={t('backup-account.image-description')}/>
```

Comment:
Does make sense to use a more specific label for the image in case to have more images on this page?